CMakeModules/build_clBLAS.cmake

```diff
@@ -0,0 +1,39 @@
+include(ExternalProject)
+
+set(prefix ${CMAKE_BINARY_DIR}/third_party/clBLAS)
+set(clBLAS_location ${prefix}/lib/import/${CMAKE_STATIC_LIBRARY_PREFIX}clBLAS${CMAKE_STATIC_LIBRARY_SUFFIX})
+if(CMAKE_VERSION VERSION_LESS 3.2 AND CMAKE_GENERATOR MATCHES "Ninja")
+  message(WARNING "Building clBLAS with Ninja has known issues with CMake older than 3.2")
+  set(byproducts)
+else()
+  set(byproducts BYPRODUCTS ${clBLAS_location})
+endif()
+
+ExternalProject_Add(
+  clBLAS-external
+  GIT_REPOSITORY https://github.com/clMathLibraries/clBLAS.git
+  GIT_TAG 47662a6ac1186c756508109d7fef8827efab4504
```

Comment:
Is this GIT_TAG necessary or can we simply use HEAD always? We would prefer having HEAD.
Also, can you change the repository URL to point to https://github.com/arrayfire/clBLAS.git

Same changes for clFFT.